### PR TITLE
Peak perf/talent page

### PR DESCRIPTION
**Pull request by gfellah45**
coupled the talent page kindly review

### CODE REVIEW

**Comment by kingflamez** [comment]
Please fix your merge conflict

File: src/assets/scss/pages/_talent.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import "../components/global/button";
```
Comment (kingflamez) [inline src/assets/scss/pages/_talent.scss:1]: Please remove

File: src/assets/scss/pages/_talent.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  text-align: center;
+  color: #1f1f1f;
+  margin-bottom: 4rem;
+}
+.button__container {
+  width: 50%;
+  margin: auto;
+  text-align: center;
+  padding: 5rem 0;
+}
```
Comment (kingflamez) [inline src/assets/scss/pages/_talent.scss:17]: Please wrap you scss file with the page name. eg 
```scss
.talent {
 &__title {

  font-family: Rubik;
  font-style: normal;
  font-weight: normal;
  font-size: 24px;
  text-align: center;
  color: #1f1f1f;
  margin-bottom: 4rem;
}
&__button__container {
  width: 50%;
  margin: auto;
  text-align: center;
  padding: 5rem 0;
}
}
```

The way it is currently written, you can alter other people's design

File: src/pages/talent.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      id: 6,
+      image:
+        "https://res.cloudinary.com/gfellah45/image/upload/v1601374966/img/pictures_tdnnso.png",
+      name: "Okoye CB",
+      title: "Software Developer",
+      socials: [
+        { twitter: "twitter/meyeneudo", github: "facebook/meyeneudoh" },
+      ],
+    },
+  ]
```
Comment (kingflamez) [inline src/pages/talent.jsx:74]: There is a data folder, please add your data there

File: src/pages/talent.jsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  return (
+    <div>
```
Comment (kingflamez) [inline src/pages/talent.jsx:76]: Add `.talent` to your parent div

**Review by kingflamez — CHANGES_REQUESTED** [review]
Made some comments, please revert

File: src/assets/scss/pages/_talent.scss (hunk 2 above)
Comment (gfellah45) [reply src/assets/scss/pages/_talent.scss]: done

File: src/assets/scss/pages/_talent.scss (hunk 1 above)
Comment (gfellah45) [reply src/assets/scss/pages/_talent.scss]: done

**Pull request merged**